content/03.results.md

```diff
@@ -246,3 +246,28 @@ If the submitter provided cell types, the submitter annotations are compared to
 A summary of this comparison is included in the cell type report along with a table summarizing the submitter cell type annotations and a UMAP plot where each cell is colored by the submitter annotation.
 The Jaccard similarity index is calculated for all pairs of cell type labels in submitter annotations and `SingleR` annotations and in submitter annotations and `CellAssign` annotations.
 The results from both comparisons are displayed in a stacked heatmap available in the report, an example of which is shown in Figure {@fig:figS5}B.
+
+## Analysis of bulk RNA-seq
+
+Several projects in the ScPCA Portal quantified many samples' expression both at the bulk and single-cell level.
+Previous research has suggested that, compared to bulk RNA-seq, single-cell RNA-seq may fail to capture certain cell types [@doi:10.1038/s41587-020-0465-8] due, for example, to technical aspects of library preparation.
+We therefore sought to ask whether we could identify differences in cell type composition between these two modalities.
```

Comment:
I would maybe soften this a little -- we didn't really directly ask about cell type composition with this analysis.